code/game/gamemodes/objective.dm

```diff
@@ -385,17 +385,18 @@ datum/objective/steal
 	var/target_name
 
 	var/global/possible_items[] = list(
-		"the captain's antique laser gun" = /obj/item/weapon/gun/energy/captain,
+		"the captain's antique revolver" = /obj/item/weapon/gun/projectile/revolver/coltsaa,
```

Comment:
I'm still generally trying to keep compatibility with glloydstation. So in this case, I think it makes more sense (in general) to add a file somewhere in the nerva folder and overwrite these definitions there. We've been moving towards having most things initialized on a per-map basis anyways, so it makes more sense for these to be done like that too.